### DISCUSSION TITLE
Update quickstart for generator-langium

### DISCUSSION THREAD

**Discussion by luan-xiaokun** [eclipse-langium/langium#988] [Ideas]
Currently the template of `generator-langium` includes a validator service demo and a cli generator demo, both of which are related to the Hello-World grammar. Users often forget to delete/comment them after changing the `.langium` grammar file and get compiling errors (at least for me).

Updating the quickstart guide in the template may help improving user experience.

**Comment by msujew** [comment]
I agree, we should keep the validation there for users who are new to Langium, but put it in a comment block so it doesn't prevent compilation after changing the grammar.

**Comment by spoenemann** [comment]
The CLI will be made optional in https://github.com/langium/langium/pull/954
